sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosItemOperationType.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.batch.BatchRequestResponseConstant;
+
+public enum CosmosItemOperationType {
+
+    Create(BatchRequestResponseConstant.OPERATION_CREATE),
```

Comment:
java enum all all upper case
Create -> CREATE.

we should follow this for the new public enum.

Reply:
Yeah. Fixed.